packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncSession.kt

```diff
@@ -172,11 +172,18 @@ public interface SyncSession {
          * The Realm is open and data will be synchronized between the device and the server
          * if the underlying connection is connected.
          *
-         * The session will remain in this state until the Realm
-         * is closed. In which case it will become [DYING].
+         * The session will remain in this state until the session is either paused or the
+         * Realm is closed. In which case it will become either [PAUSED] or [DYING].
```

Comment:
[nit] `In which case it will become either [PAUSED] or [DYING].` is not a proper sentence - you probably want to merge the two sentences together or reword the second one.